src/workflow_engine/nodes/error.py

```diff
@@ -0,0 +1,48 @@
+# workflow_engine/nodes/error.py
+
+from typing import Literal
+
+from ..core import (
+    Context,
+    Data,
+    Empty,
+    Node,
+    NodeExecutionError,
+    Params,
+)
+
+
+class ErrorInput(Data):
+    info: str
+
+
+class ErrorParams(Params):
+    error_name: str
+
+
+class ErrorNode(Node[ErrorInput, Empty, ErrorParams]):
+    """
+    A node that always raises an error.
```

Comment:
What's the point of having this node? To test the workflow error handling?

Reply:
yep that's it